ibis/mapd/operations.py

```diff
@@ -377,15 +377,15 @@ def _format_multipolygon_value(value):
 def _format_geo_metadata(op, value):
     value = copy(value)
     srid = op.args[1].srid
-    geotype = op.args[1].geotype
+    geotype = op.args[1].geotype or 'geometry'
```

Comment:
Since this will be the second place we're doing this `or` pattern, does it make sense to add `'geometry'` as the default here? https://github.com/ibis-project/ibis/blob/8427e9eef2d692d0e06500d339ae2d1de015cb00/ibis/expr/datatypes.py#L619

Reply:
Sounds good. I will check if there is any difference when geotypes is empty and when it is `geometry`

Reply:
I didn't find much about this ... not sure if we can assume that None will be geometry .. I will change the logic here.